Find a functionally equivalent code:android.graphics.ImageFormat.getBitsPerPixel:COMMENT
Method Modifier: public      static      
Comment:/**
 * Use this function to retrieve the number of bits per pixel of an
 * ImageFormat.
 *
 * @param format
 * @return the number of bits per pixel of the given format or -1 if the
 * format doesn't exist or is not supported.
 */

@@ -18,25 +18,25 @@
             return 12;
         case RAW_SENSOR:
             return 16;
-        case BAYER_RGGB:
-            return 16;
+        case RAW10:
+            return 10;
     }
     return -1;
 }

